test/functional/spec-runner/index.js

```diff
@@ -121,6 +121,21 @@ function generateTopologyTests(testSuites, testContext, filter) {
           afterEach(() => testContext.cleanupAfterSuite());
           testSuite.tests.forEach(spec => {
             it(spec.description, function () {
+              if (requires.authEnabled) {
+                // TODO: We do not have a way to determine if auth is enabled in our mocha metadata
+                // We need to do a admin.command({getCmdLineOpts: 1}) if it errors (code=13) auth is on
+                this.skip();
+              }
```

Comment:
Syncing the tests added some tests that use `authEnabled` as a runOn requirement, I believe we don't currently have a way of filtering for this, correct me if I'm wrong and I'll make the change. If not I left a comment suggesting the method the python driver uses.

Reply:
One option could be to check the `$AUTH` environment variable. At least when running on Evergreen, it'll be `auth` if the tests are running against a server with auth enabled. A bit more hassle when running tests locally since you need to set that env var when running your tests, but the simplicity of the implementation could make that worth it.

Reply:
Flagging this comment because it seems I didn't leave it as a review last time.